source/playgrounds.txt

```diff
@@ -30,12 +30,35 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+|vsce| splits your Playground window to display your
+:guilabel:`Playground Result` in the right-side pane.
+
+If |vsce| returns these results in |json| format, you may edit these
+results. If the result returns |json|\-formatted content, |vsce|
+displays it with |json| syntax highlighting and includes an
+:guilabel:`Edit Document` link at the top of the document.
+
+In the :guilabel:`Playground Result` pane, you can:
+
+- Save the results to a file.
+- Edit the results in the :guilabel:`Playground Result` pane if you
```

Comment:
Replace `Edit the results in the :guilabel:`Playground Result` pane if you made a :doc:`findOne or find query </read-document-playground>` in the Playground.` with `Use the `Edit Document` link to open a |json| document in a new pane in edit mode if you made a :doc:`findOne or find query </read-document-playground>` in the Playground.`. - something like this. You still write it like you can edit docs in the playground result pane, you can not, it is always read-only.

<img width="1552" alt="Screenshot 2021-02-18 at 20 03 37" src="https://user-images.githubusercontent.com/16307679/108409126-203b1a00-7226-11eb-8583-3213bbe6d795.png">

Reply:
And please note, that I have split the `mongodbVSCodePlaygroundDB.sales:1.json` document to the right just to demonstrate to you that this is a separate thing, not the playground result pane. It is not being split to the right automatically,